docs/user-guide/running.md

```diff
@@ -15,6 +15,55 @@ When your environment is running, you can [deploy your workload](deploying-workl
 
 To force device re-registration remove EdgeDevice CR from the cluster and `/var/local/yggdrasil/device` directory from the device
 
+---
+## Automatically
+
+### Prerequisites
```

Comment:
Do you think it makes sense to make sure that `nftables` and `podman` is installed to run the agent?

Reply:
As part of the documentation or the script?

Reply:
I thought about the script but we could be explicit in the documentation as well.